Gopkg.lock

```diff
@@ -7,6 +7,27 @@
   packages = ["quantile"]
   revision = "4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/dataverse-broker/dataverse-broker"
```

Comment:
We should find a way to remove this as a dependency.

Reply:
Who added this in the Gopkg.lock?

Reply:
This is autogenerated, so go must think we need this dependency.

Reply:
The only way is to probably to manually edit the Gopkg.toml file to remove the dependency. (https://github.com/golang/dep/issues/1470)

Reply:
I checked the toml file, no mention of our repo.